svo (priestreport).xml

```diff
@@ -17,7 +17,7 @@
 			<colorTriggerBgColor>#000000</colorTriggerBgColor>
 			<regexCodeList />
 			<regexCodePropertyList />
-			<Trigger isActive="no" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
+			<Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
```

Comment:
Did you mean to enable this group for everyone?

Reply:
it will deactivate if they are not in priest class. When I was coding, I was in the priest class and it unlocked all of these folders. Should not affect people when they update and log back in

Reply:
Hmm, ok. I've always been told to minimize lines being changed especially if irrelevant to the actual change... ah well, different strokes and all that.